matlab_simulation/05-estimation/runFastSLAM.m

```diff
@@ -0,0 +1,61 @@
+%% Fast SLAM Example
+% Author: Bismaya Sahoo.
+%% Initializations
+clear all; close all; clc;
+% Primary Initializations           
+poseInitial=[0,0,0]';%initial Pose
+dt=0.1; timeInterval=dt;%time interval
+finalTime=20;%final Time in sec
+MotionNoise=[0.001 0 0;0 0.001 0;0 0 0.0001];R=MotionNoise;
+MeasurementNoise=[0.02 0;0 0.02];Q=MeasurementNoise;
```

Comment:
Great jobs, but I would suggest a single-column and aligned coding style with spaces besides signs like

MotionNoise = [ 1, 2, 3 ;
                          4, 5, 6 ;
                          7, 8, 9 ] ;
R = MotionNoise;

MesasurementNoise = [ 1, 2, 3 ;
                                       4, 5, 6 ;
                                       7, 8, 9 ] ;
R = MotionNoise;